### ISSUE TITLE
FLIGHTGROUP issue

### ISSUE THREAD

**Issue by phr0gz** [FlightControl-Master/MOOSE#1959]
Hello,

There is an issue when you **repeat** a spawn of a Group that is using FLIGHTGROUP, see miz: FlightGroupTestRepeatStuck.miz

There is an other issue when **every of two/three spawn** the group is destroyed after take-off, see miz:FlightGroupTestDestroy.miz

Thanks
[FlightGroupTestDestroy.miz.zip](https://github.com/FlightControl-Master/MOOSE/files/11732484/FlightGroupTestDestroy.miz.zip)
[FlightGroupTestRepeatStuck.miz.zip](https://github.com/FlightControl-Master/MOOSE/files/11732485/FlightGroupTestRepeatStuck.miz.zip)

**Comment by phr0gz** [comment]
Thanks for the answer Applevangelist, It make sense, I made a mistake in the exist condition.

For the alias name I will check this evening.
But It's still strange that it is working in one miz and not in the other miz.

**Comment by phr0gz** [comment]
Ok so I changed the condition to 
```CapturePlane[captureNumber] == nil or CapturePlane[captureNumber]:IsAlive() ~= true``` and it solve my mistake. But on this one the repetition is working fine without changing the alis with random element.

For the first one as suggested I added the random element and it works.

I don't what to think because one is working without random element in the alias and the other don't and I need to add the random element in the alias. both of the function are the same the only thing that is changing is the departure airbase.

Anyway problem solved, but the behaviour is still enigmatic for me.

I added both mission without random elements in the alias, feel free to have a look. Or you/I can close this issue :), thx Applevengelist.
[FlightGroupTestDestroy.miz.zip](https://github.com/FlightControl-Master/MOOSE/files/11738405/FlightGroupTestDestroy.miz.zip)
[FlightGroupTestRepeatStuckNo random.miz.zip](https://github.com/FlightControl-Master/MOOSE/files/11738406/FlightGroupTestRepeatStuckNo.random.miz.zip)
